### ISSUE TITLE
can this work with pure react native project?

### ISSUE THREAD

**Issue by sibelius** [viewsdx/yarn-workspaces-cra-crna#29]
does this only work with cra and crna or a pure react native project should work as well?

can we add docs for this config?

**Comment by SimenB** [comment]
by copying from #26 I got it working for Expo, so I suppose normal RN works as well (didn't need `crna-make-symlinks-for-yarn-workspaces`, that just worked)

**Comment by SamMatthewsIsACommonName** [comment]
Yes every example seems to be to do with expo, however mature projects like ours have nothing to do with expo. A working config would be greatly appreciated!
I tried to run and got Error: EEXIST: file already exists for expo... and I havent defined expo as a dependency anywhere???

**Comment by sibelius** [comment]
expo has this package https://github.com/expo/expo/tree/master/packages/expo-yarn-workspaces

**Comment by SamMatthewsIsACommonName** [comment]
@sibelius thanks but is that not also for an expo project?
Edit: to be clear, our project was created with react native cli a long time ago, and isn't related to expo in any way (uses a lot of native modules etc)
